File: fflocation/fflServer/src/model/Position.java
package model;

/**
 * This class represents a position with it's own latitude, longitude and date.
 */
public class Position{


	private String latitude;
	private String longitude;
	
	private String date;
	
	public String toString(){
		return date + "(" + latitude   + "," + longitude+ ")";
	}
	
	public Position(){
		latitude=null;
		longitude=null;
		date = null;
	}
	public Position(float latitude, float longitude, String dt)
	{
		this.latitude =  String.valueOf(latitude);
		this.longitude = String.valueOf(longitude);
		this.date = dt;
	}

	public void setLatitude(float latitude) {
		this.latitude = String.valueOf(latitude);
	}
	public void setLatitude(String latitude) {
		this.latitude = (latitude);
	}

	public String getLatitude() {
		return latitude;
	}
	
	public void setLongitude(float longitude) {
		this.longitude = String.valueOf(longitude);
	}
	public void setLongitude(String longitude) {
		this.longitude = (longitude);
	}
	public String getLongitude() {
		return longitude;
	}
	public void setDate(String dt) {
		this.date = dt;
	}
	public String getDate() {
		return date;
	}
	
}

File: fflocation/fflwebsite/web_services/position.php
<?
      /**
	This class is needed so as to communicate 
	with the server side. Instead of the two sides
	sending strings they send Objects type "Position"
	for sending and getting the position of the users.
      */
      class Position{
	    private $longitude;
	    private $latitude;
	    private $date;
		
	    public function toStringPosition(){
		  return $this->date + "(" + $this->longitude + "," + $this->latitude + ")";
	    }
		
	    public function __construct(){
		  $this->longitude=null;
		  $this->latitude=null;
		  $this->date=null;
	    }

	    public function setLatitude($latitude1) {
		  $this->latitude=$latitude1;
	    }

	    public function getLatitude() {
		  return $this->latitude;
	    }
		
	    public function setLongitude($longitude1) {
		  $this->longitude=$longitude1;
	    }

	    public function getLongitude() {
		  return $this->longitude;
	    }
		
	    public function setDate($dt) {
		  $this->date=$dt;
	    }
		
	    public function getDate() {
		  return $this->date;
	    }
      }
?>